tests/drivers/spi/spi_loopback/boards/rv32m1_vega_ri5cy.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_0"
+CONFIG_SPI_LOOPBACK_SLAVE_NUMBER=2
+CONFIG_DEBUG_OPTIMIZATIONS=y
```

Comment:
is CONFIG_DEBUG_OPTIMIZATIONS needed?

Reply:
I tried removing it, but it caused a reset somewhere in the test.

Reply:
I think a bug needs to filed on this

Reply:
The commit message says 
> ... and turned on debug configurations to debug effectively and from excess code being optimized out.
Maybe Jose Manuel has an idea an can chime in? Sounds like something looks unreachable for the compiler and gets optimized out, kind of disconcerting